api/src/main/java/org/openmrs/module/locationbasedaccess/aop/PatientSearchAdviser.java

```diff
@@ -50,52 +38,7 @@ else if (method.getName().equals("getPatientByUuid")) {
 
     @Override
     public Advice getAdvice() {
-        return new PatientSearchAdvise();
+        return new PatientServiceInterceptor();
     }
 
-    private class PatientSearchAdvise implements MethodInterceptor {
```

Comment:
What this whole class removed? I see a lot of red.

Reply:
Yes, because I need to change the private PatientSearchAdvise class to public class due to access by the AOP based teston classes.So I moved to public class in another file. I just migrate that class to new file, No changes in the class content/workflow.